.github/workflows/ci.yml

```diff
@@ -42,6 +42,9 @@ jobs:
       - name: Install Dependencies
         id: deps
         run: |
+          echo 'set man-db/auto-update false' | sudo debconf-communicate >/dev/null
```

Comment:
Why not separate this step in a separate task and give it a proper name, eg: "Disable man page auto-update"?